pivot/templates/about.html

```diff
@@ -1,4 +1,5 @@
 {% extends "base.html" %}
+{% block title %}Pivot: About Pivot data and FAQ{% endblock %}
```

Comment:
Shouldn't the block title just be "About Pivot data and FAQ" since "Pivot: " is already a part of base.html template

Reply:
Yep, good catch, thanks!